docs/gestor_tareas.md

```diff
@@ -0,0 +1,51 @@
+# Gestor de Tareas
+
+## Criterios para designar el gestor de tareas
+1. La herramienta debe estar actualizada como máximo en los últimos 6 meses, de esta forma reduciremos la deuda técnica.
+2. La sintaxis debe ser lo más clara posible, para facilitar la implementación futura.
```

Comment:
De veras, no pongáis esto... La sintaxis va a estar siempre clara, y elegir uno u otro va a ser cuestión de gusto.